### PR TITLE
Podcast: add waiting for confirmation special

### PR DESCRIPTION
**Pull request by bitschmidty**
Some items to note:

- uses the existing "newsletter-references.md" include with current formatting as is w/ slight css hack to remove the "no significant news..." text as it is not relevant
- url of "/en/podcast/2023/09/waiting-for-confirmation/"
- proper rendering of the headphones icons on the /en/blog/waiting-for-confirmation/ page require some adding of newlines on 2 of the series entries

### CODE REVIEW

File: _posts/en/podcast/2023-09-04-waiting-for-confirmation-special.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+reference: /en/blog/waiting-for-confirmation/
+name: 2023-09-04-waiting-for-confirmation
+slug: 2023-09-04-waiting-for-confirmation
+type: podcast
+layout: podcast-episode
+lang: en
+---
+This podcast is a compilation of previous podcast discussions about the [Waiting
+for confirmation]({{page.reference}}) series on mempool and relay policy. Series
+coauthors Mark "Murch" Erhardt and Gloria Zhao describe the concepts in the
```
Comment (murchandamus) [inline _posts/en/podcast/2023-09-04-waiting-for-confirmation-special.md:13]: I would like to suggest mentioning Gloria first, both alphabetically and by contribution. :)

```suggestion
coauthors Gloria Zhao and Mark "Murch" Erhardt describe the concepts in the
```

File: _includes/specials/policy/en/03-bidding-for-block-space.md [hunk 2]
```diff
@@ -102,4 +102,4 @@ Wallets that support CPFP and RBF can also start with a conservative
 feerate and then update the transaction’s priority using CPFP or RBF if
 needed.
 
-[transaction size calculator]: /en/tools/calc-size
\ No newline at end of file
+[transaction size calculator]: /en/tools/calc-size
```
Comment (murchandamus) [inline _includes/specials/policy/en/03-bidding-for-block-space.md:105]: If anyone else is confused, this adds a newline at the end of the file.

**Review by murchandamus — COMMENTED** [review]
Oh that’s a very cool idea! I could see people want to listen to the entire thing in one go.

**Comment by harding** [comment]
I like this.  Thanks @bitschmidty !

**Review by murchandamus — APPROVED** [review]
ACK

**Pull request merged**